pom.xml

```diff
@@ -316,7 +316,7 @@
         <identity.application.auth.oidc.package.export.version>${project.version}
         </identity.application.auth.oidc.package.export.version>
 
-        <carbon.identity.framework.version>5.15.7</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.17.5</carbon.identity.framework.version>
```

Comment:
is this upgrade needed? if not, shall we remove it?

Reply:
Removed